### ISSUE TITLE
Have a IRC bot with loging

### ISSUE THREAD

**Issue by eschnou** [nodecopter/nodecopter.com#23]
Not directly related to the web site, but since the irc channel is starting to get active and some interesting discussion are going on, I think it would be great to have a bot and channel logs somewhere.

I don't have a running irc bot, so if anyone could help. If not, then I can try to setup one, although I've never done this.

**Comment by shama** [comment]
Check out https://github.com/nlogax/ircjsbot and https://github.com/nlogax/ircjsbot-plugins
Or https://github.com/github/hubot

**Comment by felixge** [comment]
I agree that this would be nice to have. However, unfortunately there seems to be no simple hosted IRC logging solution (I'd be happy to throw $10-20 / year at this). Given the absence of such a service I won't have time to set this up anytime soon, but I'd certainly be happy if somebody else did it.
